### ISSUE TITLE
mail module is failing in ansible 2.7.4

### ISSUE THREAD

**Issue by hryamzik** [ansible/ansible#49965]
##### SUMMARY

mail module is failing in ansible 2.7.4

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

mail

##### ANSIBLE VERSION

```paste below
2.7.4
```

##### STEPS TO REPRODUCE

Not sure if this helps as soon as behaviour may be related to SMTP server settings

```yaml
      - name: send email
        mail:
          host: smtp.example.com
          subtype: html
          subject: test
          from: notme@example.com
          to: me@example.com
          secure: never
          body: content
```

##### EXPECTED RESULTS

```
TASK [send email] ************************************************************************************************************************************************************************************************************************
changed: [test]
```

##### ACTUAL RESULTS

```
TASK [send email] ************************************************************************************************************************************************************************************************************************
The full traceback is:
Traceback (most recent call last):
  File "/tmp/ansible_mail_payload_jR5xfh/__main__.py", line 263, in main
    code, smtpmessage = smtp.connect(host, port)
  File "/usr/lib/python2.7/smtplib.py", line 317, in connect
    (code, msg) = self.getreply()
  File "/usr/lib/python2.7/smtplib.py", line 365, in getreply
    + str(e))
SMTPServerDisconnected: Connection unexpectedly closed: timed out

fatal: [openvpnas2]: FAILED! => {
    "changed": true,
    "invocation": {
        "module_args": {
            "bcc": [],
            "body": "test",
            "cc": [],
            "charset": "utf-8",
            "from": "smtp.example.com",
            "headers": [],
            "host": "smtp.example.com",
            "password": null,
            "port": 25,
            "secure": "try",
            "sender": "smtp.example.com,
            "subject": "test",
            "subtype": "html",
            "timeout": 20,
            "to": [
                "me@example.com"
            ],
            "username": null
        }
    },
    "msg": "Unable to Connect smtp.example.com:25: Connection unexpectedly closed: timed out",
    "rc": 1
}
```

Works fine on ansible 2.7.1

**Comment by sivel** [comment]
This issue was recently fixed in https://github.com/ansible/ansible/pull/49222 and subsequently included in the 2.7.5 release of Ansible.

**Comment by xoxys** [comment]
My bad.... On RHEL7 it's only fixed on 2.7.6
